pep-0007.txt

```diff
@@ -50,6 +47,11 @@ C dialect
   Future C99 features may be added to this list in the future
```

Comment:
That seems unlikely now that 3.11 uses a different standard.

Reply:
If that block were to move up to 3.11, it could be reformulated as
```
  Optional C11 features may be added to this list in the future
  depending on compiler support (mostly significantly MSVC).
```

Reply:
Removed.